src/manifest.c

```diff
@@ -233,6 +233,9 @@ static struct manifest *manifest_from_file(int version, char *component, bool he
 		if (strncmp(line, "contentsize:", 12) == 0) {
 			contentsize = strtoull(c, NULL, 10);
 		}
+		if (strncmp(line, "actions:", 8) == 0) {
+			post_update_action = strdup(c);
+		}
```

Comment:
Solid questions! This is intended for the MoM only, so we can change that to check that only if it's in the MoM. 
The latest MoM should be the one it gets the action from, so in this case it will always be doing that - implicitly if both happen to define an action. I don't think that needs extra code, so I'll update with a comment clarifying that.